### DISCUSSION TITLE
Create meeting with different media

### DISCUSSION THREAD

**Discussion by e-Feitosa** [sircharlo/meeting-media-manager#377] [Ideas] [answered]
I would like to suggest the following option for M³: It would be interesting to have an option to customize [create] other meetings during the week. For example: 
¹Superintendent's visit (on Tuesday), 
²virtual congress to broadcast by zoom, 
³ guest speaker from betel among other meetings. 

In short, it would be a possibility to create additional meetings, since if my meeting is on Saturday, and I have an event on Sunday, it no longer appears the media (example the virtual congress).

![image](https://user-images.githubusercontent.com/90984283/171265003-84f0a330-27cb-4079-8a61-74b1f67d029b.png)

**Comment by sircharlo** [comment, marked as answer]
I'm not sure I understand? 
In your screenshot, if you click on June 4th (Saturday) for example, you can add media for a "special" meeting that will take place on Saturday.
If I understood you correctly, the feature is already present, no?
Simply click on any day to manage media for that day, whether there is a regularly scheduled meeting or not.
If I misunderstood, please let me know.

**Comment by e-Feitosa** [comment]
Thanks for the clarification, really this option will help to organize an additional meeting. I am grateful for the time you spend developing this excellent program. For example, I can create an additional congregation (temporary) School for Pioneers and place the media;

**Reply by sircharlo** [reply]
Absolutely!
A great use, I hadn't thought of it but absolutely. Place the media for every day of the school and you'll have your media all set up in advance.

**Reply by e-Feitosa** [reply]
@sircharlo , Just going back on that theme, would it be possible to add the following option? [x] Schools. Why? Here the hall is used by the circuit and the media is different from congregational meetings, so this option would disable downloading the meeting media and leave every day of the week as if it were a meeting day to add the media.

![image](https://user-images.githubusercontent.com/90984283/189673496-a98e641d-6cc5-4723-acc9-f37762189d48.png)

**Reply by mtdvlpr** [reply]
Hi @e-Feitosa, so if I understand correctly. You basically don't want to set any meeting days? Every day should be empty, until you add your own media to it?

**Reply by e-Feitosa** [reply]
> Oi @e-Feitosa, então se eu entender corretamente. Você basicamente não quer marcar nenhum dia de reunião? Todo dia deve estar vazio, até que você adicione sua própria mídia a ele?

@mtdvlpr , thanks for the feedback to my message, in an additional meeting (for example: Pioneer), we do not have the need to download media from the meetings, but, we use songs, images and videos that we receive from the speakers and it is necessary to make a schedule from Monday to Saturday (it would be a special schedule). Here we have 3 congregations, and I have created a congregation named circuit (or schools) and will have school, Monday, Tuesday, Wednesday, Thursday, Friday and Saturday. So I suggested something to not download the media of the meetings, but that on the days mentioned above it would be possible to add the media.

**Reply by mtdvlpr** [reply]
I can implement that in my new version (as long as @sircharlo approves of course ).  Will be in the next release.

**Reply by e-Feitosa** [reply]
@mtdvlpr, thank you very much for your work and @sircharlo i will be very grateful if you have this release to implement this option.

**Reply by mtdvlpr** [reply]
@e-Feitosa, your request feature is coming! It ships with the version alpha.15 which is releasing soon. Let me know what you think of it!

**Reply by mtdvlpr** [reply]
Hi @e-Feitosa, just checking in to ask whether your requested feature works as expected?

**Comment by mtdvlpr** [comment]
@sircharlo, if you want, you could work on this feature to get familiar with the new codebase.

**Reply by sircharlo** [reply]
Not ready for that just yet, I'm still looking into Vue and Nuxt but I'm still way out of my league!

**Reply by mtdvlpr** [reply]
No worries. I will pick it up. Take your time to get familiar with everything.
